WeakAuras/LibSpecializationWrapper.lua

```diff
@@ -0,0 +1,85 @@
+if not WeakAuras.IsLibsOK() then return end
+if not WeakAuras.IsRetail() then return end
+local AddonName, Private = ...
+
+local LibSpec = LibStub("LibSpecialization")
+
+local nameToSpecMap = {}
+local nameToUnitMap = {
+  [GetUnitName("player", true)] = "player"
+}
+
+local subscribers = {}
+
+Private.LibSpecWrapper = {}
+if LibSpec then
+  local frame = CreateFrame("Frame")
+  frame:RegisterEvent("PLAYER_LOGIN")
+  frame:RegisterEvent("GROUP_ROSTER_UPDATE")
+  frame:SetScript("OnEvent", function()
+    local ownName = GetUnitName("player", true)
+
+    nameToUnitMap = {}
```

Comment:
shouldn't it use wipe(nameToUnitMap) for less garbage?

Reply:
Doesn't really matter, because this happens so infrequently.

Using a new table each time does show up in the memory consumption, but is likely actually faster than wiping the table.